src/pdn/src/PdnGen.tcl

```diff
@@ -179,13 +179,14 @@ proc define_pdn_grid {args} {
 sta::define_cmd_args "set_voltage_domain" {-name domain_name \
                                            [-region region_name] \
                                            -power power_net_name \
+                                           [-secondary_power secondary_power_net_name] \
```

Comment:
make sure to update the documentation to include this option

Reply:
Thanks. This is done.